packages/yew/src/functional/hooks/use_state.rs

```diff
@@ -111,6 +113,12 @@ impl<T: fmt::Debug> fmt::Debug for UseStateHandle<T> {
 }
 
 impl<T> UseStateHandle<T> {
+    /// Returns the inner value of the handle.
+    pub fn get(&self) -> Rc<T> {
+        // Safety: `UseStateReducer<T>` is `repr(transparent)` and only contains `T`
+        unsafe { transmute(self.inner.get()) }
```

Comment:
Can we avoid the transmute somehow?

Reply:
not really, this transmute converts between `Rc<UseStateReducer<T>>` and `Rc<T>`, there's not much you can do with a value inside an `Rc`

Reply:
You would need to convert `Rc<UseStateReducer<T>> -> Rc<T>` (assuming `T` is not guaranteed to be `Clone`) some other way then (maybe via `Any`?). I think using `repr(transparent)` was a brilliant idea for `Rc`s and now I hope Rust makes it like a safe trait/`Rc` function.